pkg/p2p/libp2p/internal/handshake/handshake.go

```diff
@@ -24,10 +24,16 @@ import (
 )
 
 const (
-	ProtocolName    = "handshake"
+	// ProtocolName is the text of the name of the handshake protocol.
+	ProtocolName = "handshake"
+	// ProtocolVersion is the current handshake protocol version.
 	ProtocolVersion = "1.0.0"
-	StreamName      = "handshake"
-	messageTimeout  = 5 * time.Second // maximum allowed time for a message to be read or written.
+	// StreamName is the name of the stream used for handshake purposes.
+	StreamName = "handshake"
+	// MessageTimeout is the maximum allowed time for a message to be read or written.
+	MessageTimeout = 5 * time.Second
```

Comment:
Should MessageTimeout be exported? I am not sure that this change is needed.

Reply:
Good point, unexporting.